worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/ConvertWorkerResult.java

```diff
@@ -30,10 +32,16 @@ private ConvertWorkerResult()
 
     public static void updateDocument(final Document document, final MarkupWorkerResult markupWorkerResult)
     {
-        document.getField("MARKUPWORKER_STATUS").add(markupWorkerResult.workerStatus.toString());
+        document.getField("MARKUPWORKER_STATUS").set(markupWorkerResult.workerStatus.toString());
```

Comment:
I don't think that this is right after all after seeing its effect in familyhashing worker.